easybuild/easyconfigs/p/plc/plc-3.0.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,69 @@
+easyblock = 'Waf'
+
+name = 'plc'
+version = '3.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://pla.esac.esa.int/pla/#home'
+
+description = """
+ plc is the public Planck Likelihood Code.
+ It provides C and Fortran libraries that allow 
+ users to compute the log likelihoods of the temperature, 
+ polarization, and lensing maps.  Optionally, it also provides a python version of this library, 
+ as well as tools to modify the predetermined options for some likelihoods 
+ (e.g. changing the high-ell and low-ell lmin and lmax values of the temperature).
+"""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+
+source_urls = ['https://pla.esac.esa.int/pla/aio']
+sources = ['product-action?COSMOLOGY.FILE_ID=COM_Likelihood_Code-v3.0_R3.01.tar.gz']
+checksums = ['ea641f7ba6a1cdc6b6271079b2bb70613944260e7de57d53fee2516b77d68c8d']
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('CFITSIO', '3.48'),
+    ('cURL', '7.69.1'),
+    ('astropy', '4.0.1', versionsuffix),
+]
+
+configopts = "--gcc "                                   # Do not test for gcc and only use gcc
+configopts += "--gfortran "                             # Do not test for gfortran and use gfortran
+configopts += "--lapack_install "
+configopts += "--cfitsio_prefix=${EBROOTCFITSIO} "      # location of the EB cfitsio install
+configopts += "--extra_lib=curl "                       # not having curl in extra lib will fail to build the examples
+configopts += "--extra_lib=m "                          # not having -lm will fail the build..
+configopts += "--extra_libpath=${EBROOTCURL}/lib "      # -''-
+
+unpack_options = '--strip 3'
+buildininstalldir = 'true'
+
+modextravars = {
+    'CLIK_PATH': '%(installdir)s',
+    'CLIK_DATA': 'share/clik',
+    'CLIK_PLUGIN': 'rel2015',
+}
+
+modextrapaths = {
+    'PYTHONPATH': 'lib/python/site-packages'
+}
+
+sanity_check_paths = {
+    'files': [
+        'lib/libclik_f90.so',
+        'lib/liblapack_clik.so',
+        'lib/libclik.so',
+        'lib/python/site-packages/clik/lkl_lensing.cpython-38-x86_64-linux-gnu.so',
+        'lib/python/site-packages/clik/lkl.cpython-38-x86_64-linux-gnu.so',
+        'lib/python/site-packages/clik/parametric.cpython-38-x86_64-linux-gnu.so',
+        'lib/python/site-packages/clik/rel2015.cpython-38-x86_64-linux-gnu.so',
```

Comment:
@jose-d It's probably better to just check for a non-empty `lib/python/site-packages/clik` directory, since these paths are specific to the Python version (and the CPU architecture)?

Reply:
@boegel I remember some digging around when I was doing the plc build manually, the problem reason was missing `*.so` file.. I see that it could be annoying for the future generations preparing new versions of plc, but on the other side, it could avoid some non-obvious errors.. 
Anyway, I'm OK with removing it, if you'd recommend that.